pkg/fronius/symo.go

```diff
@@ -63,6 +65,39 @@ type (
 		EnergyTotal float64 `json:"E_Total"`
 	}
 
+	symoInverterRealtime struct {
+		Body struct {
+			Data SymoInverterRealtimeData `json:"Data"`
+		}
+	}
+	SymoInverterRealtimeData struct {
+		//currents of strings 1 to 4
+		IDC1 RealTimeDataPoint `json:"IDC"`
+		IDC2 RealTimeDataPoint `json:"IDC_2"`
+		IDC3 RealTimeDataPoint `json:"IDC_3"`
+		IDC4 RealTimeDataPoint `json:"IDC_4"`
+
+		//voltages of strings 1 to 4
+		UDC1 RealTimeDataPoint `json:"UDC"`
+		UDC2 RealTimeDataPoint `json:"UDC_2"`
+		UDC3 RealTimeDataPoint `json:"UDC_3"`
+		UDC4 RealTimeDataPoint `json:"UDC_4"`
+
+		//AC frequency
+		FAC RealTimeDataPoint `json:"FAC"`
+
+		//AC power
+		PAC RealTimeDataPoint `json:"PAC"`
+
+		//total energy
+		TOTAL_ENERGY RealTimeDataPoint `json:"TOTAL_ENERGY"`
+	}
```

Comment:
The fields on this struct reads rather cryptic. If you know the meaning of abbreviations, please use the full name, e.g. `AcPower` or `AcFrequency` or `EnergyTotal`, no need to capitalize it. Also please try to document the meanings of a value if you know it (what's the meaning, or the unit). For example, I have no idea what "strings 1 to 4" means, and that's hard to maintain, document or make graphs of.

Reply:
Thanks for your review! I've adjusted the naming, added comments and also added a test with testdata from my Symo. Could you check again?